File: Javascript/1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let name = lines.shift();
let salary = Number(lines.shift());
let sales  = Number(lines.shift());
let TOTAL = salary + ((sales * 15) / 100);

console.log("TOTAL = R$ " + TOTAL.toFixed(2));

File: C/1009.c
#include <stdio.h>
 
int main() {
 
    char name[100];
    double salary,sales;

    scanf("%s", &name);
    scanf("%lf", &salary);
    scanf("%lf", &sales);
    
    printf("TOTAL = R$ %.2f\n", salary + ((sales * 15) / 100));
 
    return 0;
}